src/Orleans/Telemetry/SeverityLevel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Orleans.Runtime
+{
+    public enum SeverityLevel : byte
```

Comment:
You are right. I kept it as much similar to the ones present on Application Insights and NewRelic... The only difference from this and the existing one, is that this one has `Critical` as a more... critial level of error. Probably for messages notifying that something can crash the system for example... Will reuse the existing one.